public/lessons/react/forms_in_react.md

```diff
@@ -177,3 +177,11 @@ class Reservation extends React.Component {
 ```
 
 [MDN Computed Property Name ES6 Syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#New_notations_in_ECMAScript_2015)
+
+# Lab: Say My Name!
```

Comment:
I think this is also good, I like how you are extending previous labs and switching them up a bit